### ISSUE TITLE
pica filter --reduce chokes on white spaces in input list

### ISSUE THREAD

**Issue by mfakaehler** [deutsche-nationalbibliothek/pica-rs#441]
When applying a filter with the new "--reduce" option, the comma-seperated list of fields to reduce on, doesn't allow white spaces between the fields

```
 pica filter -s --reduce "003@, 002@" "002@.0 != 'this-is-never-a-mmatch'" dump.dat.gz | pica print
```
error: invalid reduce value

Removing the white-space in the reduce list yields the desired result.

Suggestion: Either add this requirement to the documentation and improve error message or allow for white spaces. IMHO, the latter would be preffered, as white spaces imprve the readability.

**Comment by niko2342** [comment]
As you suggested, allowing whitespaces in the reduce-list is the best solution. Feature is implemented in #442 and will be included in the next release.

**Comment by mfakaehler** [comment]
Great!
